### DISCUSSION TITLE
Is it possible to submit my Xidi configs?

### DISCUSSION THREAD

**Discussion by ViperAcidZX** [samuelgr/Xidi#58] [Q&A]
I have a couple of Xidi configuration for various older PC games, notably a few obscure and doujinsoft games, and I was wondering how I can submit these? Some of these configuration are not updated or tested with v4.2.0 yet (many were configured with Xidi v4.1.1).

**Comment by samuelgr** [comment]
Thank you for wanting to contribute some configurations! There are a couple of ways to do it. I've added more information on this process to the main `README.md` file on the [XidiGameConfigurations repo homepage](https://github.com/samuelgr/XidiGameConfigurations).
